ObjectToBeSpawned[] objects_in_new_scene
---
bool success